ducktape/tests/runner.py

```diff
@@ -73,10 +73,23 @@ def run_all_tests(self):
 
             # Create single testable unit and corresponding test result object
             self.current_test_context = test_context
+            result = TestResult(self.current_test_context)
+
+            if self.current_test_context.ignore:
+                # Skip running this test, but keep track of the fact that we ignored it
```

Comment:
I matched the `@Ignore` behavior in JUnit I think. 

Actually running the tests seemed unintuitive to me, but I guess we can see how we like this behavior in practice.

With the metadata in place, modifying the runner behavior is relatively simple.